### PR TITLE
Your scipt todo have been heard.

### PR DESCRIPTION
**Pull request by TheSenPie**
Your todo have been heard. Using AssetRepository to create new scripts from PropertyPanel.

### CODE REVIEW

File: editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java [hunk 1]
```diff
@@ -1356,34 +1356,45 @@ public void deleteRawAsset (FileHandle handle) {
...
+	/**
+	 * In the first synopsis form, the copyRawAsset utility copies the contents of the source_file to the target_file.
+	 * In the second synopsis form, the contents of each named source_file is copied to the target_file target_directory.
+	 * The names of the files themselves are changed in case of collision, while replace flag is false.
+	 * @param source_file
+	 * @param target_file
+	 * @param replace
+	 * @return FileHandle of newly copied file.
+	 */
+	public FileHandle copyRawAsset (FileHandle source_file, FileHandle target_file, boolean replace) {
```
Comment (Tom-Ski) [inline editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java:1372]: What are these underscores in var names!?

Comment (Tom-Ski) [reply editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java:1372]: As far as I can tell, this is doing nothing but changing names? Please dont do this in feature PRs, needs to be reverted

**Review by Tom-Ski — CHANGES_REQUESTED** [review]

File: editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java (hunk 1 above)
Comment (TheSenPie) [reply editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java:1372]: mb, it's the UNIX influence

Comment (TheSenPie) [reply editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java]: fixed

**Pull request merged**